mithril-signer/src/runtime/signer_services.rs

```diff
@@ -267,7 +267,7 @@ impl<'a> ServiceBuilder for ProductionServiceBuilder<'a> {
         let mithril_stake_distribution_signable_builder =
             Arc::new(MithrilStakeDistributionSignableBuilder::default());
         // TODO: 'allow_unparsable_block' parameter should be configurable and its default value set to false
```

Comment:
The comment should be updated since the default value is what's expected :).

Reply:
Done :heavy_check_mark: